### PR TITLE
Add field patch endpoint documentation

### PR DESCRIPTION
**Pull request by vitorezende**

### CODE REVIEW

**Comment by vitorezende** [comment]
[[Documentation] Add field PATCH endpoint to api documentation](https://trello.com/c/nOp5pwL9/2060-documentation-add-field-patch-endpoint-to-api-documentation)

File: docs/field_boundaries_endpoints.md [hunk 1]
```diff
@@ -323,6 +324,117 @@ A Field as a JSON object.
   </TabItem>
 </Tabs>
 
+### Update a Field
+
+&nbsp<span class="badge badge--warning">PATCH</span> `/users/{leafUserId}/fields/{id}`
+
+Update the Field by `id` for the user `leafUserId`. Request body accept the updatable field properties like `"name"` to update the field name, `"farmId"` to update the related Farm of the Field and `"geometry"`, which represents the boundaries of the
```
Comment (LuisOw) [inline docs/field_boundaries_endpoints.md:331]: I think ``The request body accepts updatable field properties like...`` sounds a little bit better

**Review by LuisOw — APPROVED** [review]
It's looking nice.

File: docs/field_boundaries_endpoints.md (hunk 1 above)
Comment (vitorezende) [reply docs/field_boundaries_endpoints.md]: Done https://github.com/Leaf-Agriculture/docs/pull/223/commits/cc6110560d2c845e0cd9eae45635c70a92f7fc43

**Pull request merged**